src/main/java/org/baeldung/security/LoggedUser.java

```diff
@@ -0,0 +1,50 @@
+package org.baeldung.security;
+
+import java.util.List;
+
+import javax.servlet.http.HttpSessionBindingEvent;
+import javax.servlet.http.HttpSessionBindingListener;
+import javax.servlet.http.HttpSessionEvent;
+import javax.servlet.http.HttpSessionListener;
+
+import org.baeldung.web.dto.LoggedUserDto;
```

Comment:
I cannot compile since LoggedUserDto is missing. I think we need to fix the import here as I see that LoggedUserDto is not used.